.github/workflows/release.yaml

```diff
@@ -1,7 +1,5 @@
 name: Release
 on:
```

Comment:
Will the adding of a cron job to `Push staging -> prod` on Wednesdays be a separate task?

Reply:
It won't be done as a cron. The build sheriff will kick it off manually every week.